src/sock/km/sock.c

```diff
@@ -19,34 +23,40 @@
 
 #include "sock.tmh"
 
-#define FNSOCK_POOL_SOCKET 'kSsF' // FsSk
-#define FNSOCK_POOL_SOCKET_SEND 'sSsF' // FsSs
-#define FNSOCK_POOL_SOCKET_RECV 'rSsF' // FsSr
+#define FNSOCK_POOL_SOCKET        'kSsF' // FsSk
```

Comment:
is it time to create a single pooltag for the entire win-net-test repo?

Reply:
Wouldn't the advantage of granularity outweigh the disadvantage of a potential collision with another tag, assuming that any issue being investigated is much more likely to be an issue in a win-net-test component rather than the component using the same pooltag (which would likely be an OS prod component)?

Reply:
This was an egregious typo! I mean to suggest creating a single pooltag.h file, for exactly the reasons you've described: make it easier to find the code for a tag, and avoid dups.

Reply:
Ok lol. Turns out we already have one, so I'll move these defines there